### ISSUE TITLE
no highlight for localized name

### ISSUE THREAD

**Issue by craftwar** [chatty/chatty#83]
tested in 0.8.4b4

**Comment by tduva** [comment]
I don't think this is strictly necessary because you can always add it to the list of Highlights yourself, just like you would other alternate names. Unless you mean that that doesn't work.

**Comment by craftwar** [comment]
I know we can do that, I talk about another problem. There is an option "Highlight own name" but it doesn't work for localized name. In my opinion, adding both username and localized name to list or checking option only.
